handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
 };
 
 Character.prototype = Object.create(Player.prototype);
 Character.prototype.constructor = Character;
 
+Character.prototype.addItem = function(newItem) {
+    for (var item in this.inventory) {
+        if (this.inventory[item].name === newItem.name) {
+            this.inventory[item].quantity += newItem.quantity;
+            return "Added new quantity to existing quantity.";
```

Comment:
You could also have ids associated with the objects.

Something like:

```javascript
market = {
  0: { id: 0, name: "Shortsword", price: 1234, weight: 36 },
  23: { id: 23, name: "Breaded Shortsword", price: 12, weight: 36 },
}
```

And then your inventory would look something like:

```javascript
inventory = {
  0: { item_id: 0, quantity: 12 },
  23: { item_id: 23, quantity: 1 },
}
```